docs/book/features/file_volume.md

```diff
@@ -13,7 +13,7 @@ In order to utilize this feature in your vSphere environment, you need to make s
 
 - Enable and configure the file service in your vSAN cluster configuration. You must configure the necessary file service domains, IP pools, network etc in order to create file share volumes. Refer to [vSphere 7.0 vSAN File service](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-82565B82-C911-42F7-85B1-E9EF973EE90C.html) documentation to get started.
 
-- Establish a dedicated file share network connecting all the kubernetes nodes and make sure this network is routable to the vSAN File Share network.  Refer to [Network Access of vSAN File Share](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-EFC00FFF-E720-44F1-B229-4C13687E6B85.html) to understand the setup better.
+- Establish a dedicated file share network connecting all the kubernetes nodes and make sure this network is routable to the vSAN File Share network.  Refer to [Network Access of vSAN File Share](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-EFC00FFF-E720-44F1-B229-4C13687E6B85.html) to understand the setup better. Configure the kubernetes nodes with the same DNS server as the one used to configure the file services in the vSAN cluster configuration.
```

Comment:
Thanks for updating the requirement for DNS server configuration in k8s node VMs. You can close https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/239 after this PR is merged.

Reply:
Yes, I marked it in the PR description.